### PR TITLE
[13.0] [IMP] github_connector: Added the SOURCE_CODE_LOCAL_PATH environment variable

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT20653

### CODE REVIEW

**Comment by CarlosRoca13** [comment]
@pedrobaeza can you check if this is the correct way to use the environment variable without break the previous option

**Comment by CarlosRoca13** [comment]
image

On the image you can see that the local_path field take the value of /tmp/... set on the .travis.yml and computed on github.repository.branch model

File: github_connector/models/github_repository_branch.py [hunk 1]
```diff
@@ -99,7 +99,10 @@ class GithubRepository(models.Model):
 
     # Init Section
     def __init__(self, pool, cr):
-        source_path = tools.config.get("source_code_local_path", False)
+        source_path = (
+            tools.config.get("source_code_local_path", "")
```
Comment (pedrobaeza) [inline github_connector/models/github_repository_branch.py]: Extract this to a method

File: github_connector/demo/github_repository_branch.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+Copyright 2018 Surekha Technologies (https://www.surekhatech.com)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+    <!-- Github organization -->
```
Comment (pedrobaeza) [inline github_connector/demo/github_repository_branch.xml]: Only branches with no demo data  for modules?

File: github_connector/demo/github_repository_branch.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (pedrobaeza) [inline github_connector/demo/github_repository_branch.xml]: The file shold be suffixed with `_demo` at the end (`github_repository_branch_demo.xml`)

File: github_connector/models/github_repository_branch.py [hunk 4]
```diff
@@ -292,7 +295,10 @@ def _compute_organization_serie_id(self):
 
     @api.depends("complete_name")
     def _compute_local_path(self):
-        source_path = tools.config.get("source_code_local_path", "")
+        source_path = (
+            tools.config.get("source_code_local_path", "")
+            or os.environ["SOURCE_CODE_LOCAL_PATH"]
```
Comment (pedrobaeza) [inline github_connector/models/github_repository_branch.py]: This one should have also more priority than config one, and this way it's going to fail if not present. Shouldn't it be?:

```suggestion
            or os.environ.get("SOURCE_CODE_LOCAL_PATH")
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: github_connector/demo/github_repository_branch.xml (hunk 2 above)
Comment (CarlosRoca13) [reply github_connector/demo/github_repository_branch.xml]: I added the module demo data on the github_connector_odoo

**Comment by CarlosRoca13** [comment]
@pedrobaeza changes done!

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Fast-tracking this for fixing other PRs:

/ocabot merge minor

**Pull request merged**